test/action/params_test.rb

```diff
@@ -153,6 +154,13 @@
       params.valid?.must_equal true
       params.errors.must_be_empty
     end
+
+    it "has input available as methods" do
```

Comment:
This is a nice to have feature, but we should get a subscriber operator (`#[]`) correct too. ;)